at_client_mobile/README.md

```diff
@@ -73,9 +73,9 @@ print(atSignsWithState); // Prints a map of atsigns with their status.
 
 /// Output:
 /// {
-///   "@atsign1": true, // @atsign1 is set as primary
-///   "@atsign2": false,
-///   "@atsign3": false
+///   "atSign1": true, // atSign1 is set as primary
+///   "atSign2": false,
+///   "atSign3": false
 /// }
```

Comment:
The '@' is required in the three change dlines above

Reply:
replaced with 91ac39a